tst/com/amazon/corretto/crypto/provider/test/EvpSignatureSpecificTest.java

```diff
@@ -460,25 +625,52 @@ public void simpleCorrectnessAllAlgorithms() throws Throwable {
             final Signature nativeSig = Signature.getInstance(algorithm, AmazonCorrettoCryptoProvider.INSTANCE);
```

Comment:
Have this use `NATIVE_PROVIDER`.

Reply:
i went ahead and changed all references to AmazonCorrettoCryptoProvider.INSTANCE to NATIVE_INSTANCE in this test file.